pkg/webui/styles/fonts.styl

```
@@ -66,5 +70,6 @@
   font-family: 'Material Icons'
   font-style: normal
   font-weight: 400
+  font-display: fallback
```

```suggestion
  font-display: block
```
For icons, I think this is the better option. It's better not showing anything than the unstyled, lengthy icon ids.